enhancements/arbiter-clusters.md

```diff
@@ -0,0 +1,692 @@
+
+### API Extensions
+
+#### Installer API Change
+
+The `installConfig` will include a `installConfig.arbiter` object for a
+`MachinePool` to configure arbiter infra structure.
+
+#### OCP Config API Change
+
+The Infrastructure config fields for `ControlPlaneTopology` will support the new
+value of `HighlyAvailableArbiter`
+
+`InfrastructureTopology` will be left as `HighlyAvailable` in this situation
+since the existing rule is we only highlight when we are dealing with
+`SingleReplica` workers. In this instance, at minimum 2 masters will be able to
+schedule worker workloads, thus follow the same pattern as a 3 Node cluster.
+This makes the most sense for now since having `InfrastructureTopology` denote a
+halfway point between single and three might be a moot distinction for workers.
+However, we might need to revisit this as a product team might assume a
+`InfrastructureTopology: HighlyAvailable` == `3 Workers`.
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+We will need to make sure that HyperShift itself can not be installed on the
+`HighlyAvailableArbiter` topology. We should also take the opportunity in that
+change to also disallow `SingleReplica`.
+
+#### Standalone Clusters
+
+This change is relevant to standalone deployments of OpenShift at the edge or
+datacenters. This enhancement specifically deals with this type of deployment.
+
+#### Single-node Deployments or MicroShift
+
+This change does not effect Single Node or MicroShift.
+
+### Implementation Details/Notes/Constraints
+
+Currently there are some behavior unknowns, we will need to test out our POC to
+validate some of the desires in this proposal. In it's current version this
+proposal is not exhaustive but will be filled out as we implement these goals.
+
+We currently expect this feature to mainly be used by `baremetal` installs, or
+specialized hardware that is built to take advantage of this type of
+configuration. In the current design we make two paths for cloud and baremetal
+installs in the installer. However, the cloud install is primarily for testing,
+this might mean that we simplify the installer changes if we are the only ones
+using cloud installs, since we can simply alter the manifests in the pipeline
+with out needing to change the installer.
+
+In the current POC, a cluster was created on AWS with 2 masters running
+`m6a.4xlarge` and 1 arbiter running a `m6a.large` instance. At idle the node is
+consuming `1.41 GiB / 7.57 GiB	0.135 cores / 2 cores`. The cluster installs and
+operates with the following containers running on the arbiter node. This seems
+very promising, but we will need to identify any other resources we might want
+to run or remove from this list. The list is small already so there might not be
+a desire to pre-optimize just yet.
+
+| Component                                                                                          | Component Outlook on Arbiter                                      |
```

Comment:
How does this list compare to a regular control plane machine? What's _not_ running on the arbiter, that would be, were it a normal control plane machine?